views/score.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Calculate a Single World Score</h1>
+<h3>Here you can enter a single word and get your score. Please, enter your word in this form:</h3>
+
+<form action="" method="post" accept-charset="utf-8">
+	<label for="word">Enter Word: </label>
+	<input type="text" name="word" value="Enter Word Here!" id="word">
```

Comment:
You can use a different attribute `placeholder` here instead of `value` which will put the words "Enter Word Here!" into the text entry by default, but when the user clicks on the text entry it will automatically disappear.